controller/konnect/ops/sdk/konnecttransitgateway.go

```diff
@@ -0,0 +1,17 @@
+package sdk
+
+import (
+	"context"
+
+	sdkkonnectcomp "github.com/Kong/sdk-konnect-go/models/components"
+	sdkkonnectops "github.com/Kong/sdk-konnect-go/models/operations"
+)
+
+// TransitGatewaysSDK is the interface for operating Konnect transit gateways.
+// REVIEW: define the transit gateway methods as a dedicated interface or add the methods to the `CloudGatewaysSDK` interface?
```

Comment:
These are all defined on `CloudGateways` type in the SDK https://github.com/Kong/sdk-konnect-go/blob/7b5878290ac230c0f1668f734af51fa6c073d3f7/cloudgateways.go#L5031 so I believe we should stick to that (given that networks and configurations already use this approach: https://github.com/Kong/gateway-operator/blob/ea346f51bd54c3c8ec3fe0dc17dd75c13c1679ea/controller/konnect/ops/sdk/dedicatedcloudgateways.go#L11-L21)

Reply:
Done. Moved the methods to `CloudGatewaysSDK` interface.